inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sink/FullSinkInfo.java

```diff
@@ -21,11 +21,11 @@
 import lombok.Data;
 
 /**
- * Sink info for Sort
+ * Full sink info - with stream
  */
 @Data
-@ApiModel("Sink info for Sort")
-public class SinkForSortDTO {
+@ApiModel("Full sink info")
+public class FullSinkInfo {
```

Comment:
FullSinkInfo is not clean to understand, try another name

Reply:
OK, change it to `SinkInfo`.